ticketutil/redmine.py

```diff
@@ -348,6 +348,7 @@ def add_attachment(self, file_name):
                 logger.error("Error attaching file {0}".format(file_name))
                 logger.error(e)
                 return self.request_result._replace(status='Failure', error_message=str(e))
+            # Does it ever happen that IOError is raised?
```

Comment:
Good catch - this was a carry over from the other ticketutil classes where add_attachment() needed to open the file. But redmine just takes the filename and doesn't actually open the file, so you're right - you can remove the `except IOError` block.

Reply:
The whole block was removed.